README.md

```diff
@@ -6,3 +6,20 @@ Nebulaworks Engineering use of [nixpkgs](https://github.com/NixOS/nixpkgs)
 
 * [Container Images](./imgs/README.md)
 * [Publishing](./PUBLISHING.md)
+* [iso Configs](./isos/README.md)
+
+## Resolving Nix Configurations
+While there are numerous ways of resolving Nix configs and derivations, one way to perform derivations in a non-linux environment is leveraging a Docker container. The general syntax for this approach is:
+```
+docker run --rm -it -v $(git rev-parse --show-toplevel):/build -w /build <dockerImage_withNix> /bin/sh -c "\
+    <insert_nix_command_1> &&\
+    <insert_nix_command_2> &&\
+    ...
+    "
+```
+replacing everything with <> to what is needed in that particular use case.
+
+The main advantages of this approach is:
```

Comment:
I think `Item 2` here is already mentioned above, `item 1` and `item 3` I dont think are worth calling out here. However I think it would be worth putting a link to the nix dockerfile that you have been using so others have a reference about which image to use with `docker run`

Reply:
I believe I called it out in the `README.md` in the `iso/README.md`, but I can also reference it here as well.